gradle.properties

```diff
@@ -0,0 +1,14 @@
+version = 0.9.0-alpha
```

Comment:
the latest proto version is `0.10.0`. Shouldn't we align with that initially, if this is going to be aligned with the proto releases?

Reply:
I figured we can release 0.9 first, remove `proto` from the SDK repo while then bumping to 0.10. How does that sound?

Reply:
I think that it probably doesn't matter to wait for the SDK, unless we're using identical maven coordinates. I don't think we *can* use identical maven coordinates, since the versions will collide in the 0.10.0 release: https://search.maven.org/artifact/io.opentelemetry/opentelemetry-proto/0.10.0/jar.

Reply:
I wouldn't wait for the SDK for anything here - I figured it's worth publishing both 0.9.0 and 0.10.0 here since it's easy, release, change one line, release once more. Having 0.9.0 would allow us to swap it in for the current `proto` module in the SDK without worry of any changes. Perhaps it's too much worry though given we only use it in tests there now.

Reply:
ok, I'm fine with that. :thu